packages/drivers/routerlicious-driver/src/test/tsconfig.cjs.json

```diff
@@ -0,0 +1,12 @@
+{
+	// This config must be used in a "type": "commonjs" environment. (Use fluid-tsc commonjs.)
+	"extends": "./tsconfig.json",
```

Comment:
This inherits the ESM `tsconfig.json` which excludes `commonjs` path; so, nothing is building/testing the refactored file/test.